src/test/java/uk/gov/dhsc/htbhf/utils/ClaimantRequestTestDataFactory.java

```diff
@@ -13,10 +15,12 @@ public static String buildClaimantRequestJson(ClaimValues claimValues) {
         //TODO MRS 20/09/2019: Move all values used to fill out form into ClaimValues so we can assert from there rather than Constants.
         String formattedClaimantDob = formatYearMonthDay(DOB_DAY, DOB_MONTH, DOB_YEAR);
         LocalDate dueDate = LocalDate.now().plusMonths(VALID_PREGNANCY_MONTH_INCREMENT);
-        String childDob = formatYearMonthDay(LocalDate.now().minusYears(1));
+        List<String> formattedChildrenDobs = claimValues.getChildrenDob().stream()
+                .map(date -> "\"" + formatYearMonthDay(date) + "\"")
```

Comment:
Couldn't you add the quotes using `String.join` below instead of needing to also do this?

Reply:
not easily